.github/workflows/jira-label.yml

```diff
@@ -13,15 +13,15 @@ name: Jira Label Mirroring
 on:
   issues:
     types: [labeled, unlabeled]
-    
+
 permissions:
   issues: read
 
 jobs:
   call-label-action:
-    uses: dbt-labs/jira-actions/.github/workflows/jira-label.yml@main
+    uses: dbt-labs/actions/.github/workflows/jira-label.yml@main
```

Comment:
Great catch converting this.  I'd love to get rid of the jira-actions repo.

The `actions` version has one extra required field of [project_key](https://github.com/dbt-labs/actions/blob/733ec2137c690c67fb7172710e59aa1f6eed33f6/.github/workflows/jira-creation.yml#L22-L24)

It will be `CT` for this repo.

See it in action for `dbt-snowflake` with a different `project_key` [here](https://github.com/dbt-labs/dbt-snowflake/blob/0e2f0ee6163e8c3843dfafde878ab76eca5da685/.github/workflows/jira-creation.yml#L24).

Reply:
Upon revisiting, we should actually just use the wrapper for the `CT` project: [jira-label-actions.yml](https://github.com/dbt-labs/actions/blob/main/.github/workflows/jira-label-actions.yml).  It default to the `CT` project.